<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeInfo</class>
 <widget class="QWidget" name="NodeInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>123</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frame_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>766</width>
         <height>339</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>766</width>
         <height>339</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="control::MwcPushButtonNormal" name="chnageNodeButton">
        <property name="geometry">
         <rect>
          <x>405</x>
          <y>297</y>
          <width>166</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Change mwc node that your wallet is connected to</string>
        </property>
        <property name="text">
         <string>Change Node</string>
        </property>
       </widget>
       <widget class="control::MwcFrameWithBorder" name="frame_3">
        <property name="geometry">
         <rect>
          <x>204</x>
          <y>0</y>
          <width>163</width>
          <height>244</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>27</x>
           <y>21</y>
           <width>106</width>
           <height>106</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources.qrc">:/img/Connections@2x.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="control::MwcLabelLarge" name="connectionsInfo">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>144</y>
           <width>103</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="control::MwcLine" name="frame_4">
         <property name="geometry">
          <rect>
           <x>39</x>
           <y>186</y>
           <width>91</width>
           <height>2</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="control::MwcLabelNormal" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>198</y>
           <width>139</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Connections</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="control::MwcFrameWithBorder" name="frame">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>0</y>
          <width>163</width>
          <height>244</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>27</x>
           <y>21</y>
           <width>106</width>
           <height>106</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources.qrc">:/img/Ready@2x.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="control::MwcLabelLarge" name="statusInfo">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>144</y>
           <width>103</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="control::MwcLine" name="frame_2">
         <property name="geometry">
          <rect>
           <x>39</x>
           <y>186</y>
           <width>91</width>
           <height>2</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="control::MwcLabelNormal" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>198</y>
           <width>139</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Status</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="control::MwcFrameWithBorder" name="frame_7">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>0</y>
          <width>163</width>
          <height>244</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>27</x>
           <y>21</y>
           <width>106</width>
           <height>106</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources.qrc">:/img/Difficulty@2x.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="control::MwcLabelLarge" name="difficultyInfo">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>144</y>
           <width>142</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="control::MwcLine" name="frame_8">
         <property name="geometry">
          <rect>
           <x>39</x>
           <y>186</y>
           <width>91</width>
           <height>2</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="control::MwcLabelNormal" name="label_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>198</y>
           <width>139</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Difficulty</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="refreshButton">
        <property name="geometry">
         <rect>
          <x>201</x>
          <y>297</y>
          <width>166</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Re-sync your wallet with a full node. Please note, resync process will take time.</string>
        </property>
        <property name="text">
         <string>Re-sync with node</string>
        </property>
       </widget>
       <widget class="control::MwcFrameWithBorder" name="frame_5">
        <property name="geometry">
         <rect>
          <x>402</x>
          <y>0</y>
          <width>163</width>
          <height>244</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>27</x>
           <y>21</y>
           <width>106</width>
           <height>106</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources.qrc">:/img/Blockchain@2x.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="control::MwcLabelLarge" name="heightInfo">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>144</y>
           <width>103</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="control::MwcLine" name="frame_6">
         <property name="geometry">
          <rect>
           <x>39</x>
           <y>186</y>
           <width>91</width>
           <height>2</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="control::MwcLabelNormal" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>198</y>
           <width>139</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Blocks</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="control::MwcLine" name="lineSeparator">
        <property name="geometry">
         <rect>
          <x>285</x>
          <y>276</y>
          <width>201</width>
          <height>2</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="warningLine">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>249</y>
          <width>760</width>
          <height>43</height>
         </rect>
        </property>
        <property name="text">
         <string>Warning line</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="bottomPlaceholder">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="control::MwcLabelNormal" name="embeddedNodeStatus">
        <property name="text">
         <string>Embedded node status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>56</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelLarge</class>
   <extends>QLabel</extends>
   <header>control/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLine</class>
   <extends>QFrame</extends>
   <header>control/MwcFrameWithBorder.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelNormal</class>
   <extends>QLabel</extends>
   <header>control/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcFrameWithBorder</class>
   <extends>QFrame</extends>
   <header>control/MwcFrameWithBorder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
